src/data/faq.json

```
@@ -2084,6 +2084,7 @@
     "glossary": {
         "glossaryHeadline": "Glossary",
         "optionalText": "",
+        "linkToGlossary": "Link to the glossary",

```

```suggestion
        "linkToGlossary": "Link to the Glossary",
```
Capitalize, since it is a section heading.

---

Changed it, thanks!